boards/arm64/fvp_baser_aemv8r/board.cmake

```diff
@@ -3,6 +3,7 @@
 
 set(SUPPORTED_EMU_PLATFORMS armfvp)
 set(ARMFVP_BIN_NAME FVP_BaseR_AEMv8R)
+set(ARMFVP_MIN_VERSION 11.16.16)
```

Comment:
Is this minimum value change something that always there, just not explicitly stated, or is this a new requirement?

Reply:
It's a new requirement

Reply:
Okay talked with the release team and will need to make sure this change gets called out in the release-notes.